packages/flutter/test/material/ink_well_test.dart

```diff
@@ -1056,7 +1056,7 @@ void main() {
   });
 
   testWidgets('When ink wells are reparented, the old parent can display splash while the new parent can not', (WidgetTester tester) async {
-    final GlobalKey innerKey = GlobalKey();
+    GlobalKey innerKey = GlobalKey();
```

Comment:
This test name does not sound like the behavior after this PR. Maybe we should revamp the test into the current behavior.

Reply:
As a result of our previous discussion, this was an outdated test, so I removed it.